src/algorithms/Algorithm_Brute_Force.py

```diff
@@ -0,0 +1,125 @@
+import random
+
+
+class AlgorithmBruteForce(object):
+
+    def __init__(self):        
+        # EMPTY_SUDOKU -- empty grid with dimension 9x9.
+        self.EMPTY_SUDOKU = [[0] * 9] * 9
+        # DIMENSION -- grid dimension expected, to load the grid from a string.
+        self.DIMENSION = 9
+
+    def conflict(self, (row1, col1), (row2, col2)):
+        """
+        Verifies if there is a conflict with given Columns.
+
+        row1 -- row to verify if there is a conflic
+        row2 -- row to verify if there is a conflic
+        col1 -- column to verify if there is a conflic
+        col2 -- column to verify if there is a conflic
+        Return -- boolean value, true = conflic false = no conflict
+        """
+        is_same_cell = row1 == row2 and col1 == col2
+        if row1 == row2:
+            return True
+        if col1 == col2:
+            return True
+        if row1 / 3 == row2 / 3 and col1 / 3 == col2 / 3 and not is_same_cell:
+            return True
+        return False
+
+    def setminus(self, list1, list2):
```

Comment:
the method names should be separated by underscores set_minus, bad variable names